person_class.rb

```diff
@@ -26,5 +28,9 @@ def correct_name
     @name
   end
 
+  def add_rentals(date, book)
+    Rental.new(date, self, book)
+  end
+
   private :of_age?
 end
```

Comment:
- [ ] Also to make your documentation more descriptive and professional, I think it would be better if you update the both the `## Show your support` and `Getting Started` sections with appropriate information. To help you fill the `Getting Started` section just refer this **[template example](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**.  
![Screenshot from 2023-04-12 13-08-39](https://user-images.githubusercontent.com/83042742/231531952-5d8775d9-d6bc-47d7-a10f-be0b39400071.png)